src/OpenFeature.Contrib.Providers.GOFeatureFlag/GoFeatureFlagProviderOptions.cs

```diff
@@ -40,6 +40,19 @@ public class GoFeatureFlagProviderOptions
         ///     (optional) ExporterMetadata are static information you can set that will be available in the
         ///     evaluation data sent to the exporter.
         /// </summary>
-        public ExporterMetadata ExporterMetadata { get; set; }
+        public ExporterMetadata ExporterMetadata { get; set; }
+
+
+        /// <summary>
+        ///     (optional) How long is an entry allowed to be cached in memory 
+        ///     Default: 60 seconds
+        /// </summary>
+        public TimeSpan? CacheMaxTTL { get; set; }
+
+        /// <summary>
+        ///     (optional) How many entries may be cached
+        ///     Default: 10000
+        /// </summary>
+        public int? CacheMaxSize { get; set; }
```

Comment:
We should be doing something similar to `Timeout` here.
```suggestion
        public int CacheMaxSize { get; set; } = 10000;
```